cpp/CMakeLists.txt

```diff
@@ -338,7 +319,7 @@ endif(USE_NVTX)
 ###################################################################################################
 # - link libraries --------------------------------------------------------------------------------
 
-target_link_libraries(cuspatial cudf rmm cudart cuda cusparse nvrtc GDAL::GDAL)
+target_link_libraries(cuspatial cudf cudart cuda cusparse nvrtc GDAL::GDAL)
```

Comment:
I think that you still want header-only libraries in `target_link_libraries`. CMake knows it's an interface only, and won't try to link librmm.

Reply:
we haven't built an interface `rmm` target within the CMake here and we aren't doing a `find_package` or `add_subdirectory` here so there's no `rmm` target to use